DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AddDelegateToGroup/DelegateAddedToGroupConfirmation.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.AddDelegateToGroup
+@model DelegateAddedToGroupConfirmationViewModel;
+
+@{
+  ViewData["Title"] = "Delegate added to group"; 
+
+  var routeParamsForIndexLink = new Dictionary<string, string?> {
+    { "groupId", Model.GroupId.ToString() },
+    { "page", "1" },
+  };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-l nhsuk-u-margin-bottom-9">
+      Delegate added to group
+    </h1>
+
+    <p class="nhsuk-body-l nhsuk-u-margin-bottom-8">
+      @(Model.DelegateName) has been added to the @(Model.GroupName)

```

Comment:
Missing the word `group` from the end.

Reply:
Added the group at the end of this string. Thanks